Modules/rat_command.py

```diff
@@ -55,6 +55,13 @@ class NameCollisionException(CommandException):
     pass
 
 
+class PrefixInvalidException(CommandException):
```

Comment:
Is this still necessary? If someone uses the wrong prefix we should ignore it akin any other message coming from the pipe